### PR TITLE
android-ndk: bump versions

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **android-ndk/all**

/cc @a4z for being the original author

---

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/android-ndk/all/test_package/conanfile.py [hunk 1]
```diff
@@ -4,7 +4,7 @@ class TestPackgeConan(ConanFile):
     settings = "os", "arch"
 
     def build(self):
-        pass #nothing to do, not warnings please
+        pass
```
Comment (a4z) [inline recipes/android-ndk/all/test_package/conanfile.py:7]: Removing the comment might trigger someone to remove the pass, and the warnings will be back

File: recipes/android-ndk/config.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
----
-    versions:
-      "r21d":
-        folder: all
-      "r22":
-        folder: all
+versions:
```
Comment (a4z) [inline recipes/android-ndk/config.yml:1]: I am not sure if it is possible to add more than 1 additional version into 1 PR

**Review by a4z — COMMENTED** [review]
Seems the CI needs a restart, but could be it crashed since there are more than 1 new / additional version in this PR?

File: recipes/android-ndk/config.yml (hunk 2 above)
Comment (SSE4) [reply recipes/android-ndk/config.yml:1]: it is possible

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Pull request merged**